### ISSUE TITLE
A11y: The Tabs component is needs higher color contrast in the block editor

### ISSUE THREAD

**Issue by taija** [BellevueCollege/mayflower-blocks#62]
When viewed in the block editor (**not** on the front end of the website), the tab area of the tab component doesn't have sufficient color contrast to meet WCAG requirements. It's meeting required levels on the front-end, however.

**Comment by TaylorYeeBC** [comment]
Still valid.
